TRACING.md

```diff
@@ -116,6 +115,21 @@ Each trace mode is sub-divided into two different types:
 | isolated | The traces only apply to the agent; after the container has been destroyed, the first span will start at agent startup and the last at agent shutdown | Observing agent lifespan. | |
 | collated | In this mode, spans are associated with their `kata-runtime` initiated counterparts. | Understanding how the runtime calls the agent. | Requires runtime tracing to be enabled in `configuration.toml` (`enable_tracing=true`). |
 
+# Agent shutdown behaviour
+ 
+As shown in [Trace modes](#trace-modes) section, the different trace modes
+change how the VM is shutdown. In normal operation, the `kata-runtime` manages
+the VM shutdown. The **only** scenario when the agent itself is responsible for
+shutting down the VM is when static tracing is enabled. In this scenario, the
+agent finishes the trace spans and actually exits. The systemd unit that
+launches the `kata-agent` detects this and then runs the `ExecStop=` commands,
+which shutdown the VM.
+
+> **Note:** that this implies that currently systemd is required for static
```

Comment:
Lines 129-131 suggested rewrite: 
**Note:** This implies that currently systemd is required for static tracing. However, it is not a hard requirement. The agent can be modified to invoke the `ExecStop=` commands itself to avoid relying on systemd to handle VM shutdown.